### PR TITLE
Add release events api

### PR DESCRIPTION
**Pull request by clayreimann**
Supersedes and closes #162

### CODE REVIEW

File: test/test.repo.js [hunk 1]
```diff
@@ -528,12 +529,15 @@ describe('Creating new Github.Repository', function() {
       });
    });
 
+   var sha;
```
Comment (AurelioDeRosa) [inline test/test.repo.js]: Can you move this variable at the top of the file?

File: test/test.repo.js [hunk 2]
```diff
@@ -578,6 +582,59 @@ describe('Creating new Github.Repository', function() {
          });
       });
    });
+
+   var releaseId;
```
Comment (AurelioDeRosa) [inline test/test.repo.js]: Same here

**Comment by AurelioDeRosa** [comment]
Apart from the two minor notes, everything is fine.

**Comment by clayreimann** [comment]
@AurelioDeRosa updated the PR

**Comment by AurelioDeRosa** [comment]
PR merged. Please can you update the README to include these new methods?

**Pull request closed without merging**